Examples/Baghdad/BaghdadData.py

```diff
@@ -1,5 +1,5 @@
 import sys
-sys.path.append("../../FDE-Tools")
+sys.path.append('C:\\Users\GJS\Downloads\FDE-Tools-master\FDE-Tools-master\Examples\Baghdad')
```

Comment:
Hard coding this path seems problematic. Let me think of a better way to do this.